isaric/parsers/ccp-uganda-v2.toml

```diff
@@ -400,10 +436,10 @@
 
 [[observation]]
   name = "diastolic_blood_pressure_mmHg"
-  date = { ref = "admissionDateHierarchy" }
-  phase = "admission"
-  value = { field = "diastol" }
-  if.all = [{ diastol = { "!=" = "" } }, { diastol = { "!=" = 0 } }]
```

Comment:
@ekamau mapping of diastol can be kept for phase = admission, but you could use the method used above as for avpu to indicate start and end date of the event, as we have no date available for this variable. Diastolic for study phase is ok as you have it